api/app/wildfire_one/schema_parsers.py

```diff
@@ -173,6 +174,168 @@ def calculate_intensity_group(hfi: float) -> int:
     return 5
 
 
+class FireBehaviourPrediction:
```

Comment:
Maybe this is my non-pythonista experience showing, but putting a class in module with a bunch of function feels weird to me. Maybe we ought to break out all the other functions into their own classes at some point.

Reply:
Having all of this is a wildfire_one.schema_parsers is already wrong. So there's a much bigger re-factor down the line to have that make sense. I'm going to say this file is a mess, and that's something we need to address, but it's outside the scope of this ticket.